boot/nuttx/main.c

```diff
@@ -114,6 +114,15 @@ int main(int argc, FAR char *argv[])
 
   syslog(LOG_INFO, "*** Booting MCUboot build %s ***\n", CONFIG_MCUBOOT_VERSION);
 
+#ifdef CONFIG_MCUBOOT_WATCHDOG
+  int ret = mcuboot_watchdog_init();
+  if (ret < 0)
+  {
+    syslog(LOG_ERR, "Unable to initialize the wdg timer\n");
```

Comment:
```suggestion
    syslog(LOG_ERR, "Unable to initialize the watchdog timer\n");
```
nit: Since this is an informative error message, I believe the full word is more suitable instead of the abbreviation.

Reply:
Fixed